data/language/dutch.txt

```diff
@@ -584,7 +584,7 @@ STR_0582    :
 STR_0583    :
 STR_0584    :
 STR_0585    :
-STR_0586    :Bootvormige karretjes lopen over achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
+STR_0586    :Bootvormige karretjes rijden over een achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
```

Comment:
Corsleutel, je kunt blijven volhouden dat 'rails' enkelvoud is, maar dat maakt het nog niet waar. Zoals Garogolun al aangeeft is 'rails' zowel in het Engels als in het Nederland meervoud, en _nooit enkelvoud_. Het enkelvoud van rails is rail. Als je naar het geheel wilt verwijzen, gebruik je _een spoor_ (Engels: _a track_) en nooit _een rails_.

Zoals ik al eerder zei: ik laat een PR met fouten niet mergen.